EDMCLogging.py

```diff
@@ -79,6 +83,24 @@ def __init__(self, logger_name: str, loglevel: int = _default_loglevel):
         self.logger_channel.setFormatter(self.logger_formatter)
         self.logger.addHandler(self.logger_channel)
 
+        ## Rotating Handler in sub-directory
+        ## We want the files in %TEMP%\{appname}\ as {logger_name}.log and rotated versions
+        ## This is {logger_name} so that EDMC.py logs to a different file.
+        logfile_rotating = pathlib.Path(tempfile.gettempdir())
+        logfile_rotating = logfile_rotating / f'{appname}'
+        try:
```

Comment:
Could use the [pathlib method](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir), `logfile_rotating.mkdir(exist_ok=True)`